README.md

```diff
@@ -112,41 +117,58 @@ the page.
 ### Constructor
 
 ```ts
+
 new KoaWebMonetization(opts: Object | void): KoaWebMonetization
 ```
 
 Create a new `KoaWebMonetization` instance.
 
 - `opts.plugin` - Supply an ILP plugin. Defaults to using Moneyd.
 - `opts.maxBalance` - The maximum balance that can be associated with any user. Defaults to `Infinity`.
-
+- `opts.receiveEndpointUrl` - The endpoint in your Koa route configuration that specifies where a user pays streams PSK packets to your site. Defaults to `/__monetizer/{id}` where `{id}` is the server generated ID (stored in the browser as a cookie).
+- `opts.cookieName` - The cookie key name for your server generated payer ID. Defaults to `__monetizer`.
+- `opts.cookieOptions` - Cookie configurations for Koa. See [Koa ctx setting cookies options](http://koajs.com/) for more details! httpOnly has to be false for the monetizer to work!
 ### Receiver
 
 ```ts
-instance.receiver(): Function
+instance.receive(): Function
 ```
 
-Returns a koa middleware for setting up Interledger payments with
+Returns a Koa middleware for setting up Interledger payments with
 [SPSP](https://github.com/sharafian/ilp-protocol-spsp) (used in Web
-Monetization).
+Monetization). Needs to be bound to an initialised instance (as shown in example above).
+
+### Client constructor options
+
+```ts
+new MonetizerClient(opts: Object | void): MonetizerClient
+```
+Creates a new `MonetizerClient` instance.
 
-The endpoint on which this is attached must contain `:id` in the path.
+- `opts.url` - The url of the server that is registering the KoaWebMonetization plugin. Defaults to `new URL(window.location).origin`
+- `opts.cookieName` - The cookie key name that will be saved in your browser. Defaults to `__monetizer`. This MUST be the same has `opts.cookieName` in the server configuration.
+- `opts.receiveEndpointUrl` - The endpoint where users of the site can start streaming packets via their browser extension or through the browser API. Defaults to `opts.url + '__monetizer/:id'` where id is the server generated payer ID. This MUST be the same has `opts.receiverEndpointUrl` in the server configuration.
 
-### Paid
+### Middleware for cookies
 
 ```ts
-instance.paid(opts: Object): Function
+WebMonetizationMiddleWare(monetizer: KoaWebMonetization)
 ```
+This middleware allows cookies to be generated (or just sent if already set) from the server to the client. It also injects the `awaitBalance` and `spend` methods described below.
 
-- `opts.price` - Function that takes koa `ctx` and returns price, or a number.
-  Specifies how many units to charge the user. Required.
-- `opts.awaitBalance` - Whether to make the HTTP call wait until the user has
-  sufficient balance. Defaults to `false`.
+### Charging users
```

Comment:
The `serveClient()` function isn't documented here (if we keep that in; see the suggestion I posted above)